pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/kdefault/enablecainection_patch.go

```diff
@@ -49,13 +50,23 @@ const injectCAPatchTemplate = `# This patch add annotation to admission webhook
 apiVersion: admissionregistration.k8s.io/{{ .Resource.Webhooks.WebhookVersion }}
 kind: MutatingWebhookConfiguration
 metadata:
+  labels:
+    app.kubernetes.io/name: mutating-webhook-configuration
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
   name: mutating-webhook-configuration
```

Comment:
I think here we would be create-by kustomize 
and part-of : k8s.io
also we can add the version -> v1

Reply:
@camilamacedo86 does this comment still apply?